vue.config.js

```diff
@@ -52,6 +60,57 @@ module.exports = {
         }
       }
     }
+    return {
+      optimization: {
+        runtimeChunk: 'single',
+        splitChunks: {
+          cacheGroups: {
+            vendor: {
+              test: /[\\/]node_modules[\\/](?!(@fortawesome|mavon-editor|v-calendar|vee-validate|vue-moment|vue-toasted))(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'vendors',
+              enforce: true
+            },
+            fortawesome: {
+              test: /[\\/]node_modules[\\/]@fortawesome(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'fortawesome',
+              enforce: true
+            },
+            mavon_editor: {
+              test: /[\\/]node_modules[\\/]mavon-editor(.[a-zA-Z0-9.\-_]+)[\\/]/,
```

Comment:
i thought that this will be deleted later? if left there, there won't be any problem right?

Reply:
No problem, if it is removed then this section can be removed as well. Even if this section is not removed, the result would be no `mavon-editor.js` chunk. @oliverrsebastian